modules/RTC/RTCUtils.js

```diff
@@ -93,6 +93,9 @@ let disableAGC = false;
 // Disables Highpass Filter
 let disableHPF = false;
 
+// Channel count used to enable stereo.
+let channelCount = null;
```

Comment:
Should I initialise this with 1, rather than null for the use case when hd audio shouldn't be enabled? (Although I think null might just get ignored)

Reply:
Please don't use a "global" here.